src/utils/shortString.ts

```diff
@@ -29,30 +56,75 @@ export function isDecimalString(str: string): boolean {
  *
  * @param {unknown} value - The value to be checked.
  * @return {boolean} - Returns true if the value is a string, false otherwise.
+ * @example
+ * ```typescript
+ * const isStringTrue: boolean = isString("Hello, world!");
+ * // isStringTrue = true
+ * 
+ * const isStringFalse: boolean = isString(123);
+ * // isStringFalse = false
+ * ```
  */
 export function isString(value: unknown): value is string {
   return typeof value === 'string';
 }
 
 /**
  * Test if value is a free-from string text, and not a hex string or number string
+ * @param val - The value to test.
+ * @returns Returns true if the value is a free-form string text, otherwise false.
+ * @example
+ * ```typescript
+ * const isTextTrue: boolean = isText("Hello, world!");
+ * // isTextTrue = true
+ * 
+ * const isTextFalse: boolean = isText("0x7572692f706963742f7433382e6a7067")
+ * // isTextFalse = false
+ * ```
  */
 export function isText(val: any) {
   return isString(val) && !isHex(val) && !isStringWholeNumber(val);
 }
 
 /**
  * Test if value is short text
+ * @param val - The value to test.
+ * @returns Returns true if the value is a short text, otherwise false.
+ * @example
+ * ```typescript
+ * const isShortTextTrue: boolean = isShortText("Hello, world!");
+ * // isShortTextTrue = true
+ * 
+ * const isShortTextFalse: boolean = isShortText(Hello, world! this is some random long string to enable you test isShortText function.)
+ * // isShortTextFalse = false
+ * ```
  */
 export const isShortText = (val: any) => isText(val) && isShortString(val);
 
 /**
  * Test if value is long text
+ * @param val - The value to test.
+ * @returns Returns true if the value is a long text, otherwise false.
+ * @example
+ * ```typescript
+ * const isLongTextTrue = isLongText("Hello, world! this is some random long string to enable you test isLongText function.")
+ * // isLongTextTrue = true
+ * 
+ * const isLongTextFalse: boolean = isLongText("Hello, world!");
+ * // isLongTextFalse = false
+ * ```
  */
 export const isLongText = (val: any) => isText(val) && !isShortString(val);
 
 /**
  * Split long text into short strings
+ * @param longStr - The long text to split.
```

Comment:
Same, Split what size of text to what size of text

Reply:
This has been clearly described